### PR TITLE
Add GameTest

### PR DESCRIPTION
**Pull request by kiliangrashoff**
This tests the Game class

< testing

### CODE REVIEW

**Comment by ghyl** [comment]
lgtm, if the build completes

File: src/main/java/nl/tudelft/ti2206/semgr8/model/Game.java [hunk 1]
```diff
@@ -18,12 +16,14 @@
...
      * Constructor that generates a new board.
+     *
+     * @param b the board with the state of the pieces
+     *          of this game
+     * @param s the score of tis game
      */
-    public Game() {
-//        board = RandomBoardGenerator.getRandomBoard();
-        board = BoardReader.read("/grids/grid_1.txt");
-        Logger.info("New board loaded.");
```
Comment (wroel) [inline src/main/java/nl/tudelft/ti2206/semgr8/model/Game.java]: Why is logger removed?

File: src/test/java/nl/tudelft/ti2206/semgr8/model/GameTest.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    /**
+     * Test that the game returns the score property.
+     */
+    @Test
+    public void testGetScoreProperty() {
+        game.getScoreProperty();
+        verify(score).getScoreProperty();
+    }
+}
```
Comment (albert-smit) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/GameTest.java]: missing newline

**Comment by albert-smit** [comment]
like #53 this is missing some javadoc

**Pull request merged**